propagate_const.h

```diff
@@ -47,29 +47,13 @@ class propagate_const {
 
   constexpr propagate_const(propagate_const&& p) = default;
 
-  template <class U, enable_if_t<!is_convertible<U&&, T>::value &&
-                                     is_constructible<T, U&&>::value,
-                                 bool> = true>
+  template <class U, class = enable_if_t<is_constructible<T, U&&>::value>>
```

Comment:
The same here. We don't want to milit functionality on all the compilers just because one is not standard compliant. 

BTW, a report of the compiler error would be welcome.